tests/e2e/client/apiproxy_bookstore_test.py

```diff
@@ -152,6 +152,20 @@ def verify_list_books(self, shelf, books):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.json().get('books', []), books)
 
+    def verify_jwt_locations(self):
+        response = self._call_http(
+            '/shelves')
+        self.assertEqual(response.status_code, 401)
+
+        response = self._call_http(
+            '/shelves', userHeaders={"Jwt-Header-Name": "Jwt-Value-Prefix {}".format(FLAGS.auth_token)})
+        self.assertEqual(response.status_code, 200)
```

Comment:
Could you add a  good token to x-google-iap-assert (one of our deffault locations) make sure it get 401, and 
add one with x-google-iap-assert to a wrong token and a good token to Authroization, to get 200

Reply:
Sorry, that PR is submitted and it is updated here https://github.com/GoogleCloudPlatform/esp-v2/pull/101